tests/tests_suite_test.go

```diff
@@ -73,6 +73,9 @@ func TestTests(t *testing.T) {
 		k8sReporter,
 	}
 	if qe_reporters.Polarion.Run {
+		if config.GinkgoConfig.ParallelTotal > 1 {
+			qe_reporters.Polarion.Filename = filepath.Join(flags.ArtifactsDir, fmt.Sprintf("polarion.junit.functest.%d.xml", config.GinkgoConfig.ParallelNode))
```

Comment:
oh this should have been `partial.polarionfunctest.%d.xml` to be consistent. I proposed the wrong string.

Reply:
Updated and sqashed. PTAL, thanks!